news/_posts/2023-07-31-playtest-20230731.md

```diff
@@ -0,0 +1,30 @@
+---
+kind: article
+title: "Playtest 20230731"
+author: "PunkPun"
+created_at: 2023-07-31 07:00 +0000
+disqus_id: "playtest-20230731"
+permalink: "/news/playtest-20230731/"
+---
+
+As summer is in full swing, we are excited to kick off the playtest series for the upcoming OpenRA release! This time around, it is all about spring cleaning: squashing bugs, implementing minor requested features, and preparing for larger reworks. But fear not! We still have a bunch of cool stuff lined up just for you! Without further ado, here are some notable changes you can expect in [Playtest 20230731]({{ '/download' | relative_url }}):
+
+* Added new Covert Ops missions for Tiberian Dawn: "Eviction Notice" and "Twist of Fate"
+* Added new Red Alert missions "Allies10b" and "Soviet13a" and the Counter-Strike mission "Mousetrap"
+* Fixed several palette remapping issues and restored player color brightness remapping
```

Comment:
This is underselling the key point here. I suggest rewording to simply "Restored ability to select darker player colours".